docs/topics_5/migration-paths.adoc

```diff
@@ -84,14 +93,16 @@ The following table describes the most common supported migration paths.
 |{icon-check}
 |{icon-check}
 |-
+|-
 
 |Spring Boot
 |-
 |-
 |{icon-check}
 |{icon-check}
-|{icon-check}
 |-
+|{icon-check}
+|{icon-check}
 
 |Any Java application
```

Comment:
I'm not crazy about the word "Any". I think this could be changed to "Java application". Your call.